src/main/kotlin/step2/Calculator.kt

```diff
@@ -0,0 +1,30 @@
+package step2
+
+import step2.Operator.Companion.getBySymbol
+
+const val INPUTS_FIRST_VALUE = 1
+const val ARRAY_SIZE = 2
+const val INTERVAL_SIZE = 2
+
+class Calculator {
+
+    fun calculate(first: Int, symbol: String, second: Int): Int = when (getBySymbol(symbol)) {
+        Operator.PLUS -> Operator.PLUS.operate(first, second)
+        Operator.MINUS -> Operator.MINUS.operate(first, second)
+        Operator.MULTIPLY -> Operator.MULTIPLY.operate(first, second)
+        Operator.DIVIDE -> Operator.DIVIDE.operate(first, second)
+    }
+
+    fun calculateString(input: String?): Int {
+        val inputs = input?.split(" ")
+        require(!inputs.isNullOrEmpty())
```

Comment:
```suggestion
        require(!input.isNullOrEmpty()) { "error message" }
        val inputs = input.split(" ")
```

처음에 input에 대한 Null check를 하고나서 split를 하는게 더 좋은 로직이 아닌가합니다.

require는 다음과 같이 에러 메시지를 줄 수 있어서 적절한 예외 메시지를 줄 수 있을 것 같네요. require를 사용할 수 있는 부분이 이 함수에서도 하나 더 보이는데, 그 부분도 한번 변경해보시길  바래요.

추가로 split에 문자열인 " "은 상수로 처리할 수 있지 않을까요?

Reply:
너무 급하게 코드를 작성하다보니 이런 부분도 놓쳐버렸네요..🥲 다음에는 좀 더 신경써서 작성해보도록 하겠습니다!